packages/end-to-end/cypress/support/selectorConstants.ts

```diff
@@ -40,7 +40,7 @@ export const ROLE_SELECTION_CONTINUE = '[data-cy=roleSelection-continue]';
 export const ROLE_SELECTION_ROLE = '[data-cy=roleSelection-role]';
 
 export const CERTIFICATION_SELECTION_CONTINUE = '[data-cy=certificationSelection-continue]';
-export const CERTIFICATION_SELECTION_TYPE = '[data-cy=certificationSelection-type]';
+export const CERTIFICATION_ORGANIC_OPTION = '[data-cy=certification_id-1]';
```

Comment:
Could we use THIRD_PARTY_ORGANIC since we did go through the trouble of updating across app 😂

Reply:
Good idea, thank you!